crates/stc_ts_file_analyzer/src/analyzer/class/mod.rs

```diff
@@ -1442,6 +1442,13 @@ impl Analyzer<'_, '_> {
                                             .report(ErrorKind::DefinedWithAccessorInSuper { span: p.key.span() }.into())
                                     }
 
+                                    if super_property.accessibility == Some(Accessibility::Private)
+                                        && p.accessibility != Some(Accessibility::Private)
+                                    {
+                                        self.storage
+                                            .report(ErrorKind::PrivatePropertyIsDifferent { span: super_ty.span() }.into())
```

Comment:
not sure if `super_ty.span()` is the best way to get the parent classes name

Reply:
It's the best way